src/components/event-card/EventCard.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+    <div
+        class="card vs-event-card"
+        data-test="vs-event-card"
+    >
+        <div class="card-body">
+            <VsRow class="vs-event-card__header">
+                <VsCol
+                    sm="12"
+                    md="9"
+                    class="p-0"
+                >
+                    <VsHeading
+                        class="vs-event-card__heading"
+                        heading-style="heading-xs"
+                        level="3"
+                        data-test="vs-event-card__heading"
+                    >
+                        <!-- @slot for the title of the event -->
+                        <slot name="event-card-header" />
+                    </VsHeading>
+                </VsCol>
+                <VsCol
+                    sm="12"
+                    md="3"
+                    class="my-auto p-0"
+                    v-if="$slots['event-card-date']"
+                    data-test="vs-event-card__date"
+                >
+                    <p class="vs-event-card__date">
+                        <!-- @slot for the event date -->
+                        <slot name="event-card-date" />
+                    </p>
+                </VsCol>
+            </VsRow>
+
+            <div>
```

Comment:
I think this outer div could be removed - best not to add too many divs for perfomance sake if they arent needed